### ISSUE TITLE
Deprecated warnings in mapping tests

### ISSUE THREAD

**Issue by daschuer** [mixxxdj/mixxx#13617]
### Bug Description

```
121: warning [0x1cb6368] ComponentContainer.applyLayer is deprecated; use ComponentContainer.setLayer instead
121: warning [0x1cb6368] script.deepMerge is deprecated; use Object.assign instead
```

I think it is: 
https://github.com/mixxxdj/mixxx/blob/a730d9ebdd3711de8ecff2ab5f6566d14ee79b56/res/controllers/Behringer-Extension-scripts.js#L1114
and 
https://github.com/mixxxdj/mixxx/blob/a730d9ebdd3711de8ecff2ab5f6566d14ee79b56/res/controllers/midi-components-0.0.js#L669

### Version

main

**Comment by Swiftb0y** [comment]
not sure thats a bug... its deprecated and not removed because mass-changing all the mappings without verifying with the real hardware is too risky.

**Comment by daschuer** [comment]
Yes it is no bug in the sense of "defect". I will remove the lable.
